### ISSUE TITLE
Sponge Either crashes or it dont work :/

### ISSUE THREAD

**Issue by Zenth815** [SpongePowered/SpongeForge#3091]
Here is my Crash Log
https://gist.github.com/Zenth815/d196e861656fa9ddc14e2ca390f04e6f

Im running the recommenced Sponge 2838 and forge version and idk how i keep crashing 
and before when i could get the server to start it would not load sponge and just the mods so i could not use my plugins, But when i just run my plugins on my server by them self's they worlk fine all togather and i get no issue. But as soon as i mix the plugins and mods togather i cant get them to work. Ive been working on trying to fix this for about 2 days now and i dont know what else to do to fix this so im going on here for some help. Also ive played with all of these mods togather on a single player also to see if it maybe the mods causing the problem. Ive also tried to fix it with the _asponge thing but that didnt work either.

Mod List
![Capture](https://user-images.githubusercontent.com/61522916/75511779-3e788c00-59bd-11ea-9798-9e14de1b608a.PNG)

**Comment by gabizou** [comment]
TLDR; Remove Farseek.

Duplicate of #3041 
[Explanation here](https://github.com/SpongePowered/SpongeForge/issues/3041#issuecomment-568955626)
#754
delvr/Farseek#16
